scripts/fetch_aic_models.py

```
@@ -0,0 +1,73 @@
+
+
+def get_credentials():
+    if os.path.exists(TOKEN_FILE) and os.path.getsize(TOKEN_FILE) > 0:
+        token = open(TOKEN_FILE).read().strip()
+        print("Auth: BuildKit secret gcp_token")
+        return Credentials(token=token), None
+    try:
+        print("Auth: attempting ADC (metadata server)")
+        creds, project = adc_default()
+        return creds, project
+    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)